Tests/KlaviyoSwiftTests/Vendor/ComposableArchitecture/TestStore.swift

```diff
@@ -1241,13 +1262,13 @@ extension TestStore where ScopedState: Equatable, Action: Equatable {
   ///     expected.
   @MainActor
   @_disfavoredOverload
-  func receive(
+    func receive(
```

Comment:
hmm will fix this

Reply:
Nit: for consistency, could we re-indent whole file with 4 spaces instead of 2? Select all (Command-A), then re-indent (Control-i)

Reply:
Yah I thought we had that set somewhere in the linting but maybe never enforced anything...

Reply:
That might make the diff hard to read.

Reply:
ahh good point